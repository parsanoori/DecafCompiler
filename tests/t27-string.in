define
define x y
"sdfakl;sdfja;sdkj drftghyulj"
"23456tyuhjklp;[:"
"x"
"xy"
xaerar
