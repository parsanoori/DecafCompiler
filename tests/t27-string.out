define
T_STRINGLITERAL "sdfakl;sdfja;sdkj drftghyulj"
T_STRINGLITERAL "23456tyuhjklp;[:
T_STRINGLITERAL "y"
T_STRINGLITERAL "xy"
T_ID xaerar
